src/clib/xtg/cube_value_ijk.c

```diff
@@ -45,6 +45,7 @@ cube_value_ijk(int i,
     ib = x_ijk2ic(i, j, k, nx, ny, nz, 0);
 
     if (ib < 0) {
+        throw_exception("Outside boundary in cube_value_ijk");
         // logger_warn("Problem in routine %s! Outside?", FU);
```

Comment:
There is one test still failing, and it is failing because of this. I could simply fix it by not throwing an exception as it is sort of handled already, however the commented out code seems to imply that this is not supposed to happen and could be a bug.

Reply:
It seems removing the exception and handling it elsewhere was the outcome. Might be a bit in line with: https://github.com/equinor/xtgeo/pull/552#issuecomment-852054458